pygen/pygen_src/isa/riscv_instr.py

```diff
@@ -188,7 +188,10 @@ def get_rand_instr(self, include_instr=[], exclude_instr=[],
         if(len(disallowed_instr) == 0):
             try:
                 if(len(include_instr) > 0):
-                    idx = random.randrange(0, len(include_instr) - 1)
+                    if(len(include_instr) == 1):
```

Comment:
nit: Please remove the `()` in `if` statements like this (with a single condition): according to PEP8, parentheses in `if` statements are redundant.

Reply:
Fixed!